examples/python/example_with_zone.py

```diff
@@ -25,16 +25,16 @@ def print_data(zone_id, connection):
     console"""
     result = connection.execute(
         # explicitly pass zone id before related data
-        select([cast(zone_id.encode('utf-8'), BYTEA), test_table]))
+        select([literal_column("'{}'".format(zone_id)), test_table]))
```

Comment:
what about to avoid manual escaping values, use sqlalchemy's engine and specify `type_` value for [literal_column](https://docs.sqlalchemy.org/en/14/core/sqlelement.html#sqlalchemy.sql.expression.literal_column)? I think that we can specify [String](https://docs.sqlalchemy.org/en/14/core/type_basics.html#sqlalchemy.types.String) type to apply escape rules for string values (according to [TypeEngine](https://docs.sqlalchemy.org/en/14/core/type_api.html#sqlalchemy.types.TypeEngine) documentation)?
if it will works, I prefer to use such approach. Because it will automatically work for mysql and all other DBs if we will add them.

Reply:
Sounds good, let me check it.

Reply:
I used sqlalchemy's `literal`, it performs escaping values inside. What do you think @Lagovas ?